community/reyesr3.md

```diff
@@ -0,0 +1,8 @@
+# <Ryan Reyes>
+
+<Short Biography>
+
+-Twitter: <>
+-Website: <>
+-GitHub: <https://github.com/reyesr3>
+-Languages: <Ruby On Rails, Javascript>
```

Comment:
Sorry we should make this more cleare, but can you remove the `<` and `>`? Also you can remove the Twitter & Website bulletpoints entirely :)